### PR TITLE
HAWQ-1056 - improve hawq check command line help

### PR DESCRIPTION
**Pull request by lisakowen**
- add several missing options
- update content for accuracy and clarity

fixes the command line help part of HAWQ-1056.

### CODE REVIEW

**Comment by edespino** [comment]
@lisakowen - Do you still need a review for this PR?  The PR and Jira have been open for a while without any activity (Sep 2016).

**Comment by lisakowen** [comment]
yes, need a review for this PR.

wasn't sure how to rerun the tests - i think the travis failure is
transient/unrelated to the changes.  also wasn't sure how to determine if
there were any tests that compared command line help output, and where i
might look to figure that out.

thanks ed.

**Comment by paul-guo-** [comment]
@radarwave

File: tools/doc/gpcheck_help [hunk 1]
```diff
@@ -135,27 +142,27 @@ OPTIONS
...
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/hdp/<version>/hadoop/
 
 
 Verify and validate the HAWQ platform settings with HDFS HA
-enabled, YARN HA enabled and Kerberos enabled:
+enabled, YARN HA enabled, and Kerberos enabled:
 
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/local/hadoop-<version>/
```
Comment (radarwave) [inline tools/doc/gpcheck_help]: We have both '/usr/hdp/<version>/hadoop/' and '/usr/local/hadoop-<version>/'. Maybe we should make them the same.

File: tools/doc/gpcheck_help [hunk 2]
```diff
@@ -135,27 +142,27 @@ OPTIONS
 EXAMPLES
 *****************************************************
 
-Verify and validate the HAWQ platform settings by 
-entering a host file and specifying the hadoop location:
+Verify and validate the HAWQ platform settings by  specifying
+a host file and identifying the full hadoop install path:
 
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/hdp/<version>/hadoop/
```
Comment (radarwave) [inline tools/doc/gpcheck_help]: We have both '/usr/hdp/<version>/hadoop/' and '/usr/local/hadoop-<version>/'. Maybe we should make them the same.

Comment (lisakowen) [inline tools/doc/gpcheck_help]: thanks for reviewing @radarwave.  i used both local and hdp in the directory paths to distinguish between a product hadoop installation and an open source version.  i agree it is kind of confusing.  i will have both examples use local.

**Comment by edespino** [comment]
One thing I would suggest is to be consistent in the use of the "<" and ">" characters used in the synopsis as the detailed option description.  As an example, the string "--config config_file" should instead be "--config <config_file>" to be consistent with the synopsis. BTW: this would make it consistent with the docs as well.

**Comment by edespino** [comment]
If you could refresh this PR with my comment above, I can review one last time and we can get this merged into master.

File: tools/doc/gpcheck_help (hunk 1 above)
Comment (paul-guo-) [inline tools/doc/gpcheck_help]: hdp should probably not be in the open source doc. I think it is fine for the change.

**Comment by lisakowen** [comment]
@edespino - thanks for reviewing.  i added <> around the option arguments  as you suggested.

**Comment by liming01** [comment]
Hi @lisakowen , please add HAWQ-1056 prefix to commit message, so that we can easily track the apache jira and related fix.

If you don't open any apache jira for doc change, then maybe a simple and help prefix can be used, eg. "Doc updated: ... "

**Comment by lisakowen** [comment]
@liming01 @wengyanqing @stanlyxiang - thank you for noting that i did not follow the appropriate PR conventions.  i will make sure to squash commits and include the JIRA number in the commit message in future PRs.

**Pull request closed without merging**